liquid-prep-app/src/app/components/my-crops/my-crops.component.html

```diff
@@ -1,73 +1,79 @@
-      </div>
-    </div>
-    <div class="horizontal" >
-      <div class="temp">
-        <span class="vertical-align-middle">{{todayWeather.minTemperature}}&deg;C</span>
-      </div>
-      <div>
-        <img class="vertical-align-middle" src="{{todayWeather.nightTime.iconImageUrl}}" width="25" height="25">
+  <app-header-title class="add-crop-header" fxFlexFill leftIconName="arrow_back" headerTitle="My Crops" rightIconName="volume_up" (click)="onHeaderClick($event)">
+  </app-header-title>
+
+
+  <div class="horizontal" style="margin-bottom: 8px">
+    <div class="imgHolder" *ngIf="!todayWeather">
+      <!-- <mat-icon class="vertical-align-middle primary-blue" *ngIf="loading">hourglass_empty</mat-icon> -->
+      <div  *ngIf="!loading" (click)="showError(errorMessage)">
+        <mat-icon class="vertical-align-middle alert-color">error</mat-icon>
       </div>
     </div>
   </div>
-</div>
 
-<div *ngIf="myCropStatus == 'crop-selected'">
+  <div *ngIf="myCropStatus == 'crop-selected'">
 
-  <button mat-mini-fab class="md-fab-bottom-right" (click)="fabClicked()">
-    <mat-icon>add</mat-icon>
-  </button>
+    <mat-toolbar class="add-crop-button mat-elevation-z4 custom-toolbar">
+      <mat-toolbar-row style="padding: 0 5px;">
+        <button class="mat-title toolbar-title button" (click)="fabClicked()">
+          ADD A NEW CROP
+        </button>
+      </mat-toolbar-row>
+    </mat-toolbar>
 
-
-  <mat-grid-list class="grid" cols="2" rowHeight="1:1" gutterSize="2px" *ngIf="activeTab == tabs[0]">
-    <mat-grid-tile *ngFor="let crop of myCrops">
-      <mat-card class="card" style="padding: 0px">
-        <img mat-card-image class="cardImage" (click)="onViewCropAdvice(crop)" src="{{ crop.url }}" alt="{{ crop.cropName }}" />
-
-        <div fxLayout="row"
-             fxFill
-             fxLayoutGap="28px"
-             fxLayoutAlign="space-between end"
-             style="padding-left: 8px">
-          <p class="mat-subheader">{{crop.cropName}}</p>
-          <div >
-            <button mat-icon-button [matMenuTriggerFor]="menu">
-              <mat-icon>more_vert</mat-icon>
-            </button>
-            <mat-menu #menu="matMenu">
-              <button mat-menu-item (click)="onRemoveCrop(crop)">
+    <mat-grid-list class="grid" cols="1" rowHeight="120" gutterSize="25px" *ngIf="activeTab == tabs[0]">
+      <mat-grid-tile *ngFor="let crop of myCrops" class="shadowBox">
+        <mat-card class="card full-card">
+          <div class="cardDetails">
+            <div>
+              <span class="mat-h1">{{crop.cropName}}</span>
+            </div>
+            <div class="cropDates">
+              <span>{{crop.seedingDate}} | Last Irrigated July 21</span>
+            </div>
+            <div>
+              <button class="measureButton" mat-icon-button (click)="onViewCropAdvice(crop)">
+                MEASURE
+              </button>
+              <button mat-icon-button (click)="onViewCropAdvice(crop)">
```

Comment:
we can leave the "edit button click" as TODO since the design hasn't been finished yet